c++/src/kj/async-win32.h

```diff
@@ -33,7 +33,7 @@
 
 // Include windows.h as lean as possible. (If you need more of the Windows API for your app,
 // #include windows.h yourself before including this header.)
-#define WIN32_LEAN_AND_MEAN 1
+#include "win32-api-version.h"
 #define NOSERVICE 1
```

Comment:
Maybe we should throw these into win32-api-version.h as well?

Reply:
NOMCX & NOIME too?

Reply:
Found a few more similar symbols since it seems like maybe you would prefer it all be centralized.